protobuf/go.mod

```diff
@@ -17,8 +17,8 @@ require (
 )
 
 require (
-	golang.org/x/net v0.10.0 // indirect
-	golang.org/x/sys v0.8.0 // indirect
-	golang.org/x/text v0.9.0 // indirect
+	golang.org/x/net v0.17.0 // indirect
```

Comment:
https://security.snyk.io/vuln/SNYK-GOLANG-GOLANGORGXCRYPTOSSH-6130669
```
golang.org/x/net@v0.17.0 › golang.org/x/crypto@v0.14.0 
```

Snyk failure cause

Reply:
Upgraded to golang.org/x/net@v0.20.0 uses [golang.org/x/crypto v0.18.0](https://github.com/golang/net/blob/v0.20.0/go.mod#L6), still snyk fails. Am I missing something?

Reply:
> Upgraded to golang.org/x/net@v0.20.0 uses [golang.org/x/crypto v0.18.0](https://github.com/golang/net/blob/v0.20.0/go.mod#L6), still snyk fails. Am I missing something?

Solved!
Could you provide a bit more insight on `security/snyk (nyrahul)` ?

Reply:
https://security.snyk.io/vuln/SNYK-GOLANG-GITHUBCOMGOJOSEGOJOSE-6070736
Snyk failure cause

Reply:
@Aryan-sharma11 I test all our go.mod files using snyk CLI. Other than `golang.org/x/net/http2@0.20.0`(low risk vuln, fix yet to implement upstream) I can't find any vulnerability. Also, [github.com/go-jose/go-jose](https://security.snyk.io/vuln/SNYK-GOLANG-GITHUBCOMGOJOSEGOJOSE-6070736) is a transitive dependency to a package. Can we ignore this?